client/index.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import GenDesc from './app.jsx';
+import GenDesc from './components/App.jsx';
+// const servicePort = 6001;
+// ${servicePort}
 
-fetch(
-  `http://localhost:3001/houses/${Math.floor(Math.random() * 98)}`
-)
-  .then(res => res.json())
+const houseId = window.location.pathname;
+fetch(`http://localhost:6001${houseId}}`)
```

Comment:
If your server's port is different for `DEV` and `TEST` environments, you can import your config here, and `process.env.NODE_ENV` can be referenced to decide what the port goes here.

Reply:
Also, I think line 8 is missing a `/` between `6001` and `houseId`.

Reply:
Hey, weirdly enough, the way the url is captured clientside actually includes the '/', so I have to take it away here. I'm leaving it the way it is, but that is a **good** catch.

Reply:
I did do the port environment variable for now though...